src/python/foglamp/device/server.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import asyncio
+
+from foglamp.device.coap import controller as coap_controller
+
+__author__    = "Terris Linenbach"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__   = "Apache 2.0"
+__version__   = "${VERSION}"
+
+
+def start():
+    """Starts the device service"""
+    coap_controller.start()
+    asyncio.get_event_loop().run_forever()
```

Comment:
"asyncio.get_event_loop().run_forever()" appears twice, once each in admin_api/server.py and device/server.py. I think, it should appear in device/server.py only.